src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java

```diff
@@ -363,6 +360,12 @@ public PravegaStorageDriver(
 			.findFirst()
 			.orElseGet(this::getBasicCredentialsIfAny);
 		Loggers.MSG.info("Authentication type: {}", cred == null ? "none" : cred.getAuthenticationType());
+		val connConfig = nodeConfig.configVal("conn");
+		val poolingFlag = connConfig.boolVal("pooling");
+		evtWriterConfig = EventWriterConfig
+				.builder()
+				.enableConnectionPooling(poolingFlag)
+				.build();
```

Comment:
is it intentional not to include backOffMillis and retryAttempts?

Reply:
Yep, they're missing in mongoose config, so the defaults should be used

Reply:
Do we want them in the config? Are those expected to be varied?

Reply:
Not sure that they're needed now. We will add them if needed.